test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -1028,3 +1035,69 @@ def test_core_package_version(image):
     assert (
         not violation_data
     ), f"Few packages violate the core_package specifications: {violation_data}"
+
+
+@pytest.mark.model("N/A")
+def test_package_version_regression_in_image(image):
+    """
+    This test verifies if the python package versions in the already released image are not being downgraded/deleted in the
+    new released. This test would be skipped for images whose BuildSpec does not have `latest_release_tag` or `release_repository`
+    keys in the buildspec - as these keys are used to extract the released image uri. Additionally, if the image is not already
```

Comment:
`test_package_version_regression_in_image` seems to be passing on PT 2.1 inference even though the buildspec doesn't have the `latest_release_tag` or `release_repository` keys.